selfdrive/boardd/boardd.cc

```diff
@@ -57,6 +57,8 @@ bool loopback_can = false;
 cereal::HealthData::HwType hw_type = cereal::HealthData::HwType::UNKNOWN;
 bool is_pigeon = false;
 const uint32_t NO_IGNITION_CNT_MAX = 2 * 60 * 60 * 24 * 3;  // turn off charge after 3 days
+const uint32_t VBATT_START_CHARGING = 12000;
```

Comment:
as discussed, let's make this 11.5V. Very sporadic on-off is fine. Want to make sure EONs are charging for some low voltage supply in desk setups

Reply:
If your desk setup is not able to put out 11.5v you should get a new one. Change agreed.